packages/runtime/runtime-definitions/src/summary.ts

```diff
@@ -19,6 +19,11 @@ export interface ISummaryStats {
     totalBlobSize: number;
 }
 
+export interface IGeneratedSummaryStats extends ISummaryStats{
```

Comment:
You should move this definition into container runtime since its not used anywhere else.